Packs/Stamus/Integrations/Stamus/Stamus.py

```diff
@@ -0,0 +1,398 @@
+                item_data.pop(FIELDS_SUBSTITUTION_DICT[key])
+            else:
+                item_data.pop(key)
+    return host_info
+
+
+def fetch_host_id(client: Client, args: dict[str, Any]) -> CommandResults:
+    """Fetch host_id info from an IP
+    """
+    result = client.get_host_id(args['ip'])
+    host_info = linearize_host_id(result)
+
+    table = tableToMarkdown('Host Insight', host_info, headers=['timestamp', 'ip', 'type', 'value'])
+
+    return get_command_results(result, table)
+
+
+def fetch_incidents(client: Client, timestamp: int) -> tuple[dict[str, int], list[dict]]:
+    """Fetch last alerts and build new incident structs
+    """
+
+    incidents: list[dict] = []
+    next_run = {'timestamp': timestamp}
+
+    for idx, alert in enumerate(client.get_incidents(timestamp)):
+        incident = {
+            'name': '%s_incident_%s' % (alert['target'], idx),
+            # Stamus ID used to get related events
+            'dbotMirrorId': str(alert['id']),
+            'details': alert['threat_description'],
+            'occurred': alert['timestamp'],
+            'rawJSON': json.dumps(alert),
+            'type': alert['target_type'],
+        }
+        incidents.append(incident)
+
+        timestamp = int(datetime.strptime(alert['timestamp'], DATE_FORMAT).timestamp())
+        if timestamp > next_run['timestamp']:
+            next_run['timestamp'] = timestamp
+
+    return next_run, incidents
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        message = 'ok'
+        client.test_connection()
+
+    except Exception as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):  # TODO: make sure you capture authentication errors
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+# TODO: ADD additional command functions that translate XSOAR inputs/outputs to Client
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    params = demisto.params()
+    api_token = params.get('credentials', {}).get('password')
+    server_url = params.get('url')
+    proxy = params.get('proxy', False)
+    verify_certificate = not params.get('insecure', False)
+
+    first_fetch_time = arg_to_datetime(
+        arg=params.get('first_fetch', '3 days'),
+        arg_name='First fetch time',
+        required=True
+    )
+    if first_fetch_time is None:
+        first_fetch_time = datetime.now()
```

Comment:
```suggestion
```

can it be `Nonez` looks like according to line 341 it cannot cause you have default value.